### DISCUSSION TITLE
Sciview can't display the image

### DISCUSSION THREAD

**Discussion by Beiyu-kk** [petercorke/robotics-toolbox-python#264] [Q&A]
Hi,guys,when i run this code in my pycharm,i can't see the image in my sciview,but others can.It confuse me for three days.Pls help me!
this is my code.

import roboticstoolbox as rtb
from spatialmath import SE3
robot = rtb.models.DH.Panda()
print(robot)

T = robot.fkine(robot.qz)  # forward kinematics
print(T)

T = SE3(0.7, 0.2, 0.1) * SE3.OA([0, 1, 0], [0, 0, -1])
sol = robot.ikine_LM(T)         # solve IK
print(sol)
robot = rtb.models.DH.Panda()
print(robot)
T = robot.fkine(robot.qz)  # forward kinematics
print(T)

T = SE3(0.7, 0.2, 0.1) * SE3.OA([0, 1, 0], [0, 0, -1])
sol = robot.ikine_LM(T)
# solve IK
print(sol)

q_pickup = sol.q
print(robot.fkine(q_pickup))
# FK shows that desired end-effector pose was achieved
[
![1](https://user-images.githubusercontent.com/72841628/132940687-935c2caf-1d5d-4f2e-a514-4e058c98f524.png)
![1](https://user-images.githubusercontent.com/72841628/132940693-039b4662-894b-49f5-9910-c25debbeba45.png)
](url)
qt = rtb.jtraj(robot.qz, q_pickup, 50)
robot.plot(qt.q, movie='panda1.gif')

I uplord  the photo which show my pycharm

**Comment by njkds** [comment]
I have the same problem. Have you solved it?

**Comment by njkds** [comment]
When I run it in terminal, it works!!!

**Reply by Beiyu-kk** [reply]
When i run it in the pycharm community edition, it works.

**Comment by Beiyu-kk** [comment]
But i can't run it in the pycharm professional edition

**Reply by njkds** [reply]
Me, too

**Reply by petercorke** [reply]
`matplotlib` is problematic in environments other than terminal or regular Jupyter client.  I don't have an answer but it is a frustration, we can only lean on PyCharm and matplotlib developers.
